src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -434,16 +436,17 @@ auto_flash(xrt_core::device_collection& deviceCollection)
 
     // Perform DSA and BMC updating
     for (auto& p : boardsToUpdate) {
-      bool reboot;
+      bool reboot, warm_reboot;
```

Comment:
warning: variable `reboot` is not initialized [cppcoreguidelines-init-variables]

```cpp
bool reboot, warm_reboot;
     ^
            = 0
```